composer.json

```diff
@@ -10,9 +10,9 @@
         }
     ],
     "require": {
-        "php": "^7.1 || ^8.0",
+        "php": "^7.3 || ^8.0",
         "slevomat/coding-standard": "^6.3",
-        "symplify/easy-coding-standard": "^7.3 || ^8.1"
+        "symplify/easy-coding-standard": "^7.3 || ^8.1 || ^9.0"
```

Comment:
What do you think about dropping ^7.3 and ^8.1 and just supporting ^9.0? This PR would require a major release anyway, so we might cut down on supported packages.

Reply:
I saw many people running into an ECS error on Slack and I have it myself:

```
$ vendor/bin/ecs check src/ tests/
 [WARNING] You are using YAML format in "easy-coding-standard.yml" config.      
           It is deprecated and will be removed in next ECS 9. Switch to PHP    
           format as soon as possible with                                      
           "https://github.com/migrify/config-transformer"                      
 [ERROR] in_array() expects parameter 2 to be array, null given  
```

It seems not going to be fixed in the ECS version that we provide, so nobody will be able to get around this on older technology. This even happens on a plain project with only this requirement:

> "sylius-labs/coding-standard": "^3.2"

But also on Sylius I've seen it happen on Slack.

If the ECS version is supporting anything from PHP 7.3, then I think we can be good only going with `^9.0`. Users need to make a config file inclusion change etc. anyway, so it might be ok to do.

Let me know if you want me to adjust it.

Reply:
IMHO we can just supporting Ecs ^9.0 now 👍🏻